addons/assigngear/functions/fnc_assignGear.sqf

```diff
@@ -99,49 +99,64 @@ _unit setUnitLoadout (configFile >> 'EmptyLoadout');
                     };
                 } forEach _x;
             };
-            case 12: { // linkedItems
+            case IDX_LINKED_ITEMS: { // linkedItems
                 {_unit addWeapon _x} forEach _x;
             };
-            case 13: { // primaryWeapon
+            case IDX_PRIMARY_WEAPON: { // primaryWeapon
                 private _weapon = selectRandom _x;
                 if !(_weapon isEqualTo '') then {_unit addWeapon _weapon};
             };
-            case 14: { // scope
+            case IDX_PRIMARY_MAGAZINE: { // primarymagazine
+                {
+                    if !(_x isEqualTo '') then {_unit addPrimaryWeaponItem _x};
+                } forEach _x;
```

Comment:
Not quite sure this is correct. It doesn't add the UGL from example loadout to the primary.

Reply:
`addPrimaryWeaponItem` works fine for adding UGL's in game but not from assignGear it seems.

Reply:
It does, just didn't update the correct loadout file 👍